packages/core/src/payment/payment-method-action-creator.ts

```diff
@@ -40,11 +44,14 @@ export default class PaymentMethodActionCreator {
     }
 
     @cachableAction
-    loadPaymentMethod(methodId: string, options?: RequestOptions & ActionOptions): Observable<LoadPaymentMethodAction> {
-        return Observable.create((observer: Observer<LoadPaymentMethodAction>) => {
+    loadPaymentMethod(methodId: string, options?: RequestOptions & ActionOptions): ThunkAction<LoadPaymentMethodAction, InternalCheckoutSelectors> {
+        return store => Observable.create((observer: Observer<LoadPaymentMethodAction>) => {
+            const state = store.getState();
+            const checkout = state.checkout.getCheckoutOrThrow();
+
             observer.next(createAction(PaymentMethodActionType.LoadPaymentMethodRequested, undefined, { methodId }));
 
-            this._requestSender.loadPaymentMethod(methodId, options)
+            this._requestSender.loadPaymentMethod(checkout.id, methodId, options)
```

Comment:
Think we can put `checkout.id` at the end as an optional param. As a result, no changes to the previous code are required.